tests/js/harness.js

```diff
@@ -7,10 +7,15 @@ require("gpii-express");
 require("../../");
 
 
-fluid.defaults("gpii.pouch.tests.harness", {
+fluid.defaults("gpii.test.pouch.harness", {
     gradeNames: ["fluid.component"],
-    port:       6789,
-    baseUrl:    "http://localhost:6789/",
+    databases: {
```

Comment:
In addition there needs to be a gpii.tests.pouch.loadTestingSupport() directive of the familiar form which loads these from the outside. As an additional courtesy this should in turn call gpii.express.loadTestingSupport().